### ISSUE TITLE
AppEntryType doesn't deserialize to Rust properly

### ISSUE THREAD

**Issue by ddd-mtl** [holochain/holochain-client-js#134]
AppEntryType as defined in client-js doesn't seem to deserialize properly to rust.
A simple test with a zome function:
```
#[hdk_extern]
pub fn echo_app_entry_type(entry_type: AppEntryType) -> ExternResult<()> {
   debug!("echo_app_entry_type() called: {:?}", entry_type);
   Ok(())
}
```
and typescript ui code:
```
const aet: AppEntryType = {id: 0, zome_id: 0, visibility: 'Public'};
this.callZome('echo_app_entry_type', aet);
```

Gives me this error:
```
ERROR wasm_trace [...] output_type = "core::result::Result<(), holochain_wasmer_common::result::WasmError>"; bytes = [131, 162, 105, 100, 0, 167, 122, 111, 109, 101, 95, 105, 100, 0, 170, 118, 105, 115, 105, 98, 105, 108, 105, 116, 121, 166, 80, 117, 98, 108, 105, 99]; Deserialize("wrong msgpack marker FixStr(6)")
```

**Comment by ddd-mtl** [comment]
(using holochain 0.0.165 and "@holochain/client": "^0.9.2",)

**Comment by jost-s** [comment]
fixed by #136
